custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -109,7 +109,15 @@ def aggregation_query(self):
             ('date_last_private_admit', 'person_list.date_last_private_admit '),
             ('date_return_private', 'person_list.date_return_private'),
             ('out_of_school_status', 'person_list.is_oos'),
-            ('last_class_attended_ever', 'person_list.last_class_attended_ever')
+            ('last_class_attended_ever', 'person_list.last_class_attended_ever'),
+            ('has_bank_account', 'CASE WHEN person_list.bank_account_number is not null THEN 1 ELSE 0 END'),
```

Comment:
if this field is derived from `bank_account_number` then may not need to add it as a column in the table. It could be directly calculated in the view.

Reply:
was following has_aadhar. CHanged it now